NumericOutcome.md

```diff
@@ -0,0 +1,453 @@
+# Numeric Outcome DLCs
+
+## Introduction
+
+When dealing with enumerated outcomes, DLCs require a single nonce and Contract Execution
+Transactions (CETs) are claimed using a single oracle signature.
+This scheme results in DLCs which contain a unique CET for every possible outcome, which is
+only feasible if the number of possible outcomes is of manageable size. 
+
+When an outcome can be any of a large range of numbers, then using a simple enumeration of
+all possible numbers in this range is unwieldy.
+We optimize by using numeric decomposition in which the oracle signs each digit of the outcome
+individually so that many possible outcomes can be compressed into a single CET by ignoring certain signatures.
+There will be as many nonces as there are possible digits required and CETs are claimed using
+some number of these signatures, not necessarily all of them.
+If not all signatures are used, then this CET corresponds to all events which agree on the digits for which
+signatures are used and may have any value at all other digits for which signatures are ignored.
```

Comment:
IMHO this paragraph is very hard to read and the concept is already explained in the oracle spec that is under PR (and for which @Christewart is even adding more detailed example), so I would just remove that and reference the oracle spec.

Reply:
To be completely honest I think that it doesn't make much sense for these kinds of concerns to be included in the oracle spec for anything other than motivation whereas here they are necessary so I think it should be the other way around.

That said, I am more than happy to re-do the paragraph to make it more readable!